yapapi/runner/__init__.py

```diff
@@ -438,30 +434,45 @@ async def fill_work_q():
         # Py38: find_offers_task.set_name('find_offers_task')
         try:
             task_fill_q = loop.create_task(fill_work_q())
-            services = {
+            get_done_task: Optional[asyncio.Task[Task[D, R]]] = None
```

Comment:
This (asyncio) task just removes a single completed (yapapi) task from `done_queue`.